serverless-logic/serverless-logic-getting-started/serverless-logic-deploying-workflows.adoc

```diff
@@ -0,0 +1,47 @@
+:_content-type: ASSEMBLY
+[id="serverless-logic-deploying-workflows"]
+= Deploying workflows
+:context: serverless-logic-deploying-workflows
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+You can deploy the {ServerlessLogicProductShortName} workflows on the cluster in two modes: 
+
+* Dev mode
+* Preview mode
+
+include::modules/serverless-logic-deploying-workflows-dev-mode.adoc[leveloffset=+1]
+
+[id="serverless-logic-deploying-workflows-preview-mode_{context}"]
+== Deploying workflows in preview mode
+
+You can deploy your local workflow on OpenShift in preview mode. This allows you to experiment and modify your workflow directly on the cluster, seeing changes almost immediately. Preview mode is used for final testing and validation before deploying to production. It also ensures that workflows will run smoothly in a production-like setting.
+
+.Prerequisites
+
+* {ServerlessLogicOperatorName} is installed on your cluster.
+* You have created a {ServerlessLogicProductName} project or have access to a project with the appropriate roles and permissions to create applications and other workloads in OpenShift Container Platform.
+* You have created your {ServerlessLogicProductName} project
+* Install the OpenShift CLI `(oc)`.
+
+To deploy a workflow in preview mode, {ServerlessLogicOperatorName} uses the build system on OpenShift that will automatically create the image for executing your workflow.
+
+The following sections explains how to build and deploy your workflow on a cluster using the {ServerlessLogicOperatorName} with a `SonataFlow` custom resource.
+
+include::modules/serverless-logic-workflow-base-builder-image.adoc[leveloffset=+2]
+include::modules/serverless-logic-workflow-base-builder-dockerfile.adoc[leveloffset=+2]
+include::modules/serverless-logic-workflow-changing-resource-requirements.adoc[leveloffset=+2]
+include::modules/serverless-logic-workflow-passing-arguments-internal-builder.adoc[leveloffset=+2]
+include::modules/serverless-logic-workflow-env-variables-internal-builder.adoc[leveloffset=+2]
+include::modules/serverless-logic-workflow-changing-base-builder-image.adoc[leveloffset=+2]
```

Comment:
It will be: 
[id="serverless-logic-configuring-workflows-preview-mode_{context}"]
=== Configuring workflows in Preview mode

![image](https://github.com/openshift/openshift-docs/assets/43639538/c2576f9d-0ee6-4a85-aebd-ad4c1830a8d2)